developer-docs-site/docs/concepts/coin-and-token/index.md

```diff
@@ -1,35 +1,33 @@
 ---
-title: "Follow Aptos Token Standard"
+title: "Aptos Standards"
```

Comment:
Just noting this breaks the active verb model I attempted to instantiate. It provides context for the user on what action is expected of them. Oh well.

Reply:
Woops, I'll clean these up with the big title PR I'm going to do.

I've read through more of it and now I get it.  So, I would change it now to something like:

Follow the Aptos Standards